pkg/melange/melange.go

```diff
@@ -143,8 +143,15 @@ func ReadAllPackagesFromRepo(ctx context.Context, dir string) (map[string]*Packa
 		if err != nil {
 			return p, fmt.Errorf("failed to read package config %s: %w", fi, err)
 		}
-
-		p[packageConfig.Package.Name] = &Packages{
+		
+		name := packageConfig.Package.Name
+		
+		// check that the package config name is unique
+		_, exists := p[name]
+		if exists {
+			return p, fmt.Errorf("Package config names must be unique. Found duplicate '%s'", name)
```

Comment:
It might be helpful to include the file names for the duplicate packages in the error message so it's easier to fix